data/strings/types_strings.txt

```diff
@@ -4767,6 +4767,64 @@
     comment = A weather shelter by the public transport stop.
     ref = type.amenity.shelter
 
+  [type.amenity.shelter.basic_hut]
+    comment = Compared to wilderness_hut its smaller and simpler, without a fireplace.
+    en = Bivouac Hut
+    af = bivak hut
+    ar = كوخ إقامة مؤقتة
+    az = bivouac daxması
```

Comment:
Please Capitalize all of the translations

Reply:
Ok, capitalized all translations

Reply:
Shouldn't our translate.py script provide capitalized translations automatically? @map-per how did you create translations?

Reply:
I created them with translate.py, but used a non capitalized string ("bivouac hut") as a starting point. I did that, because DeepL interpreted some capitalized words as names.

Reply:
Just checked: If I use the capitalized "Bivouac Hut" as a starting point the translations are capitalized as well.